client/lib/purchases/index.js

```diff
@@ -101,6 +105,14 @@ function isRefundable( purchase ) {
 	return purchase.isRefundable;
 }
 
+function isRemovable( purchase ) {
+	if (  isIncludedWithPlan( purchase ) ) {
+		return false;
+	}
+
+	return ( isExpired( purchase ) && purchase.isCancelable );
```

Comment:
We display only message at the moment, I think we can remove `purchase.isCancelable` check. I wanted to keep it similar to `/my-upgrades`, but as we can't cancel manually expired purchases we need to wait with this until v2.